interp/constrextern.ml

```diff
@@ -978,6 +978,11 @@ let rec extern inctx ?impargs scopes vars r =
   | GVar id when entry_has_global (fst scopes) || entry_has_ident (fst scopes) ->
       CAst.make ?loc (extern_var ?loc id)
 
+ | GInt i when Coqlib.has_ref "num.int63.wrap_int" ->
+     let wrap = Coqlib.lib_ref "num.int63.wrap_int" in
+     let r' = DAst.make (GApp (DAst.make (GRef (wrap, None)), [r])) in
+     extern_notations inctx scopes vars None r'
+
```

Comment:
And then remove the `| GInt i ...` from the `No_match` case? Also, should the `r` be replaced with `r'` ahead?
(I'm afraid I still don't understand this piece of code...)

Reply:
Sorry, I was unclear. The idea, when `r` is a `GInt` is to replace it by `wrap_int r` before calling the notation mechanism (`extern_notations`), if no notation is found (for instance when they are all deactivated by `Set Printing All`), it will raise `No_match` and things should proceed with `r` unmodified (the default printer will be run below).

Reply:
Ok, I believe the error doesn't happen anymore after my latest commit. But please check if this was what you had in mind.

Reply:
Looks good